src/dev/build/tasks/os_packages/create_os_package_kibana_yml.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { readFileSync, writeFileSync } from 'fs';
+import { resolve } from 'path';
+import { Build, Config, mkdirp } from '../../lib';
+
+export async function createOSPackageKibanaYML(config: Config, build: Build) {
+  const configReadPath = config.resolveFromRepo('config', 'kibana.yml');
+  const configWriteDir = config.resolveFromRepo('build', 'os_packages', 'config');
+  const configWritePath = resolve(configWriteDir, 'kibana.yml');
+
+  await mkdirp(configWriteDir);
+
+  let kibanaYML = readFileSync(configReadPath, { encoding: 'utf8' });
+
+  [
```

Comment:
I'm iffy on this approach, but it does work.  I looked into templating kibana.yml while preserving it's relatively static nature.  Turning it into a template would remove validity of the yml, leading to more duplication.

Any thoughts here?